platform/mellanox/sdk.mk

```diff
@@ -1,6 +1,6 @@
 MLNX_SDK_BASE_PATH = $(PLATFORM_PATH)/sdk-src/sx-kernel/Switch-SDK-drivers/bin/
 MLNX_SDK_PKG_BASE_PATH = $(MLNX_SDK_BASE_PATH)/$(BLDENV)/
-MLNX_SDK_VERSION = 4.4.3326
+MLNX_SDK_VERSION = 4.4.3334
```

Comment:
should we take 4.5.xxxx eventually?

Reply:
Yes. But for Debain 11/kernel 5.10 validation we use 4.4.3334